docs/tutorial/tutorial_index.rst

```diff
@@ -0,0 +1,31 @@
+Reservoir Modelling: Bifröst Developers: RESQML Resqpy
+======================================================
+
+
+**These pages are currently under development.** However, the
+material so far in place should be usable.
+
+This page contains tutorials for programmers wanting to use the
+**resqpy** RESQML application programming interface (API).
+RESQML is the standard format for storing subsurface models. The
+resqpy API is a python library providing classes and functions
+allowing application code to work with RESQML datasets at a
+high level.
+
+The tutorials are in the form of wiki pages which include code
+snippets.  Whilst reading through a tutorial, it is recommended
+that you have a python interpreter (or Jupyter notebook) open
+in another window, so that code snippets can be cut, pasted and
+executed whilst progressing through the material.
+
+::toctree::
+        :maxdepth: 1
+        :caption: Tutorials
+
+        intro_to_resqml
+        getting_started_with_model
+        creating_test
+        working_with_coord
+        high_level_objects
+        cellular_grids
+        grid_properties
```

Comment:
I think it would make sense to move this section to the existing table of contents in `docs/index.rst`, L22-26 . 

Nb I think you have to be careful with indentation with these RST directives. I think 3 spaces is the norm